### PR TITLE
Package comby.1.2.2

### PR DESCRIPTION
**Pull request by rvantonder**
### `comby.1.2.2`
A tool for structural code search and replace that supports ~every language
Comby is a tool designed to make it easy to match change syntax for basically
every language. You can think of it like grep or sed, but with added ability to
recognize code structures like blocks and expressions (e.g., delimited by
braces or parentheses), strings (e.g., delimited by quotes), and so on.

---
* Homepage: https://github.com/comby-tools/comby
* Source repo: git+https://github.com/comby-tools/comby.git
* Bug tracker: https://github.com/comby-tools/comby/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/comby/comby.1.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "ppxlib"
+  "ppx_deriving"
+  "ppx_deriving_yojson" {>= "3.6.0"}
+  "pcre"
+  "shell"
+  "toml" {>= "6.0.0"}
+  "tls"
+  "bisect_ppx" {with-test & dev & >= "2.5.0"}
+]
+x-ci-accept-failures: ["oraclelinux-8"] # libev not available by default
```
Comment (rvantonder) [inline packages/comby/comby.1.2.2/opam:49]: I looked at some of the other `opam` files and I think adding this makes sense? Seemed like previous `oraclelinux-8` failures happened due to missing `libev` on the system but I see no reason it wouldn't work if `libev` were installed.

Comment (rvantonder) [reply packages/comby/comby.1.2.2/opam:49]: > oraclelinux-8,  comby.1.2.2 (failed: The compilation of conf-libev failed at "/usr/bin/sh ./build.sh".)

whelp, I guess that didn't work

Comment (avsm) [reply packages/comby/comby.1.2.2/opam:49]: Does adding `conf-libev` to the dependency list not work?

Comment (rvantonder) [reply packages/comby/comby.1.2.2/opam:49]: 🤦 how did I miss this

Comment (rvantonder) [reply packages/comby/comby.1.2.2/opam:50]: Updated, thanks for catching @avsm

Comment (rvantonder) [reply packages/comby/comby.1.2.2/opam:50]: whoops, I should remove the `x-ci-accept-failures` part too

Comment (avsm) [reply packages/comby/comby.1.2.2/opam]: easy to miss, there are many odd quirks with handling external dependencies across so many distributions. Glad it works now!

**Comment by rvantonder** [comment]
Adding `conf-libev` seems to not work for `oraclelinux-8`:

> #=== ERROR while compiling conf-libev.4-11 ====================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.11.2 | file:///src
# path                 ~/.opam/4.11/.opam-switch/build/conf-libev.4-11
# command              /usr/bin/sh ./build.sh
# exit-code            1
# env-file             ~/.opam/log/conf-libev-13-8e6b2b.env
# output-file          ~/.opam/log/conf-libev-13-8e6b2b.out
### output ###
# testing for libev: ............................. unavailable
# 
#       The following recquired C libraries are missing: libev.

Since the `x-ci-accept-failures: ["oraclelinux-8"]` trick didn't' work I'm just going to set it to not available

**Comment by rvantonder** [comment]
@kit-ty-kate I added `available: [ os-distribution != "oraclelinux" ]` but still seeing CI failures for it:

```
distributions
  oraclelinux-8
   comby.1.2.2 (failed: The compilation of conf-libev failed at "/usr/bin/sh ./build.sh".)
```

Any tips to skip this?

**Comment by kit-ty-kate** [comment]
I fixed the `conf-libev` package instead. Thanks!

**Comment by rvantonder** [comment]
🎉

**Comment by aryx** [comment]
yeah!

**Pull request merged**